wdio.conf.ts

```diff
@@ -0,0 +1,41 @@
+import type { Options } from '@wdio/types'
+export const config: Options.Testrunner = {
+    runner: 'local',
+    tsConfigPath: './tsconfig.json',
+    specs: [
+        './test/specs/**/*.ts'
+    ],
+    suites: {
+        authorization: [
+            './test/specs/authorization.ts'
+        ]
+    },
+    exclude: [
+        // 'path/to/excluded/files'
+    ],
+    maxInstances: 10,
+    capabilities: [
+        {
+            browserName: 'chrome',
+            'goog:chromeOptions': {
+                args: ['--disable-infobars', '--window-size=1920,1080']
+            }
+        }
+    ],
+    logLevel: 'info',
+    bail: 0,
+    waitforTimeout: 10000,
+    connectionRetryTimeout: 120000,
+    connectionRetryCount: 3,
+    framework: 'mocha',
+    reporters: [['allure', {outputDir: 'allure-results'}]],
```

Comment:
@Anastas11aZakharova Please, add the 'spec' reporter as well, just for yours repo.

Reply:
Fixed in the latest commit